src/MyElefant.php

```diff
@@ -1,239 +1,296 @@
-<?php
-namespace myelefant;
-use Dotenv;
+<?php namespace myelefant;
+
+/**
+ * MyElefant plugin
+ * To sending sms with myElefant service
+ * https://myelefant.com/
+ */
+
 use GuzzleHttp\Client;
 use Monolog\Logger;
 use Monolog\Handler\StreamHandler;
-use Monolog\Handler\FirePHPHandler;
-use Symfony\Component\Yaml\Yaml;
 use DateTime;
 use Exception;
+
 class MyElefant
 {
 
     /**
+     * Error log
      * @var logger
      */
     private $error;
 
     /**
+     * Info log
      * @var logger
      */
     private $info;
 
+
     /**
+     * Token
      * @var string
      */
     private $token;
 
     /**
-     * @var array
-     */
-    private $dotenv;
-
-    /**
-     * @var array
-     */
-    private $yamlDatas;
-
-    /**
-     * @var array
+     * Initialize plugin
+     * @var array|null $config Plugin's config
+     * @throws @Exception
+     * @return void
      */
-    private $contacts;
-
-    public function __construct(){
-        $this->yamlDatas = Yaml::parseFile(__DIR__.'/../MyElefant.yaml');
-        $this->dotenv = $this->initDotEnv();
-        $this->token = $this->getAuthentification(getenv('SECRET_KEY'));
-
-        if ($this->yamlDatas['APP_ENV'] == 'dev' ) {
-            $this->error = $this->initLogger('error',__DIR__.'/../log/error.log');
-            $this->info = $this->initLogger('info',__DIR__.'/../log/info.log');
+    public function __construct($config)
+    {
+
+        if (isset($config['debug']) === true) {
+            $this->error = $this->initLogger(
+                'error',
+                'var/log/error.log'
+            );
+            $this->info  = $this->initLogger(
+                'info',
+                'var/log/info.log'
```

Comment:
can `var/log/error.log` be configurable through the $config object?
same for `info`?